src/locales/en_US.js

```diff
@@ -130,7 +130,7 @@ const strings = {
   settings_button_change_password: 'Change Password',
   settings_button_debug: 'Debug',
   settings_button_lock_settings: 'Tap to Lock Account Settings',
-  settings_button_password_recovery: 'Password Recovery',
+  settings_button_password_recovery: 'Setup Password Recovery',
```

Comment:
You're gonna hate me for this but technically shouldn't it be "Set Up Password Recovery"? "Setup" is a noun, whereas "Set Up" is a verb / action.